### PR TITLE
Add aged/blind/disabled handling

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
The eligibility for aged/blind/disabled medicaid is federally defined [in the US Code here](https://www.law.cornell.edu/uscode/text/42/1382c#a), but the income criteria is devolved ([this table](https://www.kff.org/medicaid/state-indicator/medicaid-eligibility-through-the-aged-blind-disabled-pathway/?currentTimeframe=0&sortModel=%7B%22colId%22:%22Location%22,%22sort%22:%22asc%22%7D) seems like the best source).

### CODE REVIEW

File: openfisca_us/parameters/hhs/medicaid/aged_or_disabled/asset_limit.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+description: Maximum assets to qualify for aged/blind/disabled Medicaid coverage.
+metadata:
+  unit: currency-USD
+  name: medicaid_disabled_asset_limit
+  label: Medicaid disabled asset limit
```
Comment (MaxGhenis) [inline openfisca_us/parameters/hhs/medicaid/aged_or_disabled/asset_limit.yaml:5]: reference?

File: openfisca_us/parameters/hhs/medicaid/benefit/total_population.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
+description: Total population by state.
```
Comment (MaxGhenis) [inline openfisca_us/parameters/hhs/medicaid/benefit/total_population.yaml:1]: I think this should go in a calibration parameter folder

(also, reference)

File: openfisca_us/variables/gov/hhs/medicaid/eligibility/aged_blind_disabled/meets_medicaid_disabled_asset_test.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+class meets_medicaid_disabled_asset_test(Variable):
+    value_type = bool
+    entity = Person
+    label = "Meets Medicaid disabled asset test"
+    unit = USD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        assets = person("spm_unit_assets", period)
```
Comment (MaxGhenis) [inline openfisca_us/variables/gov/hhs/medicaid/eligibility/aged_blind_disabled/meets_medicaid_disabled_asset_test.py:12]: This is a person-level field?

File: openfisca_us/variables/gov/hhs/medicaid/eligibility/is_medicaid_eligible.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    value_type = bool
+    entity = Person
+    label = "Meets Medicaid financial criteria"
+    unit = USD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        person_type = person("medicaid_person_type", period)
+        person_types = person_type
+        return select(
```
Comment (MaxGhenis) [inline openfisca_us/variables/gov/hhs/medicaid/eligibility/is_medicaid_eligible.py:14]: Could be a `where`

File: openfisca_us/variables/gov/hhs/medicaid/medicaid.py [hunk 5]
```diff
@@ -3,9 +3,19 @@
...
+    documentation = "Estimated benefit amount from Medicaid (average amount per person from State expenditure)."
     label = "Medicaid benefit"
     unit = USD
     reference = "https://www.law.cornell.edu/uscode/text/42/1396a"
+
+    def formula(person, period, parameters):
+        eligible = person("is_medicaid_eligible", period)
+        state_code = person.spm_unit("state_code_str", period)
+        medicaid = parameters(period).hhs.medicaid
+        average_payment_per_person = (
```
Comment (MaxGhenis) [inline openfisca_us/variables/gov/hhs/medicaid/medicaid.py:17]: I'd suggest making this its own variable

**Review by MaxGhenis — COMMENTED** [review]
Imagine I'm jumping the gun on a review here but some thoughts

**Comment by nikhilwoodruff** [comment]
To do:
- [ ] Add individual/partnership entity (@MaxGhenis any better ideas for a name other than IndividualOrPartnership?)
- [ ] Add remaining details for asset tests and income deductions for disabled Medicaid

**Pull request merged**